_sips/minutes/2017-05-08-sip-minutes.md

```diff
@@ -0,0 +1,93 @@
+
+### [SIP-33 - Match infix & prefix types to meet expression rules](http://docs.scala-lang.org/sips/make-types-behave-like-expressions.html)
+
+Skipped since they are waiting for the prototype from the author of the proposal.
+
+### Scala Library Changes
+ [YouTube time 10:52](https://youtu.be/6rKa4OV7GfM?t=652)
+
+ **Adriaan** starts presentation and notes that feedbacks on his proposal are available through the 2.13 platform. It is more reorganization of things in different modules. He suggests list of packages, from the ticket, that he believes shouldn’t be in the core:
+ *Scala concurrent*, *Scala.ref*, *Scala.cis*, *Scala.compat* (that is already totally deprecated), *Scala.text* (that has already couple of things that are deprecated), *Scala.util*; whereas *Scala.io* and *Scala.sys* are good candidates for replacements with better community modules.
+ Also, some hashing could be removed in separate package and make Scala package cut clean. He is open to discuss if some of these packages should stay in the Scala Library Jar. Most of these packages are subjects of deprecations.
+ Scala concurrent could live in its current form, but it could be split out since it’s dependent platform however.
+
+ **Jorge** asks what they should change in XML. **Adriaan** says that it’s already a model and all these packages should wing as Scala changes. If no one from the community does push forward to maintain that, they should continue maintaining them through depreciation cycle, helped by replacement through SPPs. All these packages and classes of packages should be available in 2.13 as Jars, but you should add them by yourself to your classpath if you are using them.
+
+ **Martin** asks what they’ll get with this breaking of the system, since each package depends on each other.
+
+ **Adriaan** propose that Scala concurrent can stay, if there is no use of separation. He also agrees that Scala.util should not be removed. But others are good candidates for separation, due to their lower quality. That will allow them faster cycle of reparation.
+
+ **Martin** says to keep Scala math.
+
+ **Sebastian** also suggests that Scala.ref also should rest.
+
+ **Adriaan** said that it was just a proposal of maximal list of packages that they could split out and he’s ready to put it down to the more reasonable size. (Adriaan sent a document). The document shows the list of parts that are dependable and how platform would work if they’d be split out.
+
+ **Sebastian** believes that these platforms should not be split out for the moment.
+
+ **Adriaan** agrees with that. He believes that is probably too painful to change anything about it.
+
+ **Jorge** proposes to keep thinking on this and to discuss it in the next two months, migration cycle.  
+
+ **Seth** said that 2.13 circle progresses will bring more information and feedback about the usage. He believes that it is a good start to begin with and they’ll know more as they go through it.
```

Comment:
"circle progress" not sure what this means.

Reply:
"said that as the 2.13 cycle progresses"